en/5-roadmap.md

```diff
@@ -65,23 +62,22 @@ Our roadmap would follow the priority order below:
 - Enable Deeds renting
 - Xeed creation
 - Badges improvement and credentials development
-- DeFi services – in the wallet, credit cards partnership
+- DeFi services – in the wallet, credit card partnership
 
-We are counting on the early adopters from the DAOs and tech-savvy companies during the building stage. We should also have some early adopters from traditional companies that gain immediate value from the WoM by providing relevant engagement programs to their people, on top of their current traditional compensation plans.
+During the building stage, we are counting on the early adopters from the DAOs and tech-savvy companies. We should also have some early adopters from traditional companies that gain immediate value from the WoM by providing relevant engagement programs to their people on top of their current conventional compensation plans.
 
-MEED minting spans a 20 year period. By that time, we aspire to have achieved building a mature ecosystem, having spread to many organizations and many users globally.
+MEED minting spans 20 years. By then, we aspire to have built a mature ecosystem, spreading it to many organizations and workers globally.
```

Comment:
"MEDD has been minting for 20 years. Our aspiration is to develop a mature ecosystem and extend its reach to numerous organizations and workers worldwide."